lightly/models/mlp.py

```diff
@@ -0,0 +1,15 @@
+from torch import nn
+
+
```

Comment:
#57 will also introduce utility classes for the projection heads and predictor heads.